tools/travis_before_install.sh

```diff
@@ -27,20 +27,24 @@ retry () {
     return 0
 }
 
+# speed up installation on 2.6, 2.7, 3.3 and 3.4
+if [ $TRAVIS_PYTHON_VERSION == 2.6 -o \
+     $TRAVIS_PYTHON_VERSION == 2.7 -o \
+     $TRAVIS_PYTHON_VERSION == 3.3 -o \
+     $TRAVIS_PYTHON_VERSION == 3.4 ]; then
+# use conda env
+echo jej
+fi
+
 
-# on Python 2.7, use the system versions of numpy, scipy, and matplotlib
-# and the minimum version of cython and networkx
-if [[ $TRAVIS_PYTHON_VERSION == 2.7* ]]; then
-    virtualenv --system-site-packages ~/venv
-    sudo apt-get install python-scipy python-matplotlib python-imaging
-    sed -i 's/cython>=/cython==/g' requirements.txt
-    sed -i 's/networkx>=/networkx==/g' requirements.txt
-    sed -i '/pillow/d' requirements.txt
-else
-    virtualenv -p python --system-site-packages ~/venv
```

Comment:
Do you know what was the magic sauce, venv, pillow, scipy or matplotlib?

System pillow and scipy are not meeting the requirements, should we lower the requirements? If the tests pass, I guess this means we require pillow>=1.1.7 and scipy>=0.9.0.